client_tools.py

```diff
@@ -119,7 +120,7 @@ def main(args):
             PicturaeUndoBatch(MD5=md5_insert)
     elif args.subcommand == 'update':
         if args.collection == "Botany_PIC":
-            pic_config = read_json_config(collection="Botany_PIC")
+            pic_config = initialize_collection_config(collection="picbatch")
```

Comment:
"collection" is a bit of a misnomer when talking about "picsbatch" - let's change this to "section", (category?  something that isn't collection)